src/components/ContactList.tsx

```diff
@@ -46,7 +46,11 @@ export default function ContactList(props) {
     contactListErrorBottomSheet,
     setContactListErrorBottomSheet,
   ] = useState(React.createRef());
-  const selectectcontactlist = props.selectedContacts
+  const [
+    contactPermissionBottomSheet,
+    setContactPermissionBottomSheet,
+  ] = useState(React.createRef());
+  const selectedcontactlist = props.selectedContacts
```

Comment:
`const contactPermissionBottomSheetRef = useRef<BottomSheet>()` would remove the need for `useState` and also give the ref a well-defined type.